src/client/components/Pagination/Pagination.component.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import './Pagination.styles.css';
```

Comment:
Here you have a check failure, it needs am empty line after import statement not followed by another import.

Reply:
done